tests/testthat/test-2merge.R

```diff
@@ -102,11 +102,14 @@ test_that("merge", {
     merged  <- mergeRows(tse, f=rowData(tse)$group2, mergeTree=TRUE)
     merged2 <- mergeRows(tse, f = rowData(tse)$group2, mergeTree = FALSE)
     merged3 <- mergeRows(esophagus, f = rowData(esophagus)$group2, mergeTree = TRUE)
+    merged4 <- .merge_features(tse, merge.by = rowData(tse)$group2)
+    merged5 <- mergeRows(tse, f = rowData(tse)$group2)
```

Comment:
How about trying to pass e.g. `mergeTree` parameter? You could add `mergeTree = TRUE` here to ensure that `...` is working

Reply:
ack